recipes/dammit/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+gem install crb-blast
```

Comment:
I think it will be, as the `ruby` is a $PREFIX version. I will double-check to make sure.